### ISSUE TITLE
Move this repo to JuliaHealth?

### ISSUE THREAD

**Issue by DilumAluthge** [notZaki/DICOM_Dictionary.jl#2]
What do you think about moving this repo to JuliaHealth so it can live alongside DICOM.jl?

cc: @notZaki

**Comment by notZaki** [comment]
Yes, this repo along with [DICOMSamples](https://github.com/notZaki/DICOMSamples) could be moved to JuliaHealth. I'll try converting the latter from a script into a package.

I also need to pick a consistent naming scheme (`DICOMDictionary` vs `DICOM_Dictionary`). Do you have a preference?

**Comment by DilumAluthge** [comment]
Most packages do the `PascalCase` thing, so I'd lean towards that. But either works.
